lib/CfTemplateGenerators/S3.js

```diff
@@ -0,0 +1,18 @@
+function replaceS3BucketFunctionWithAlias(bucket, functionAlias, functionName) {
+  const lambdaConfigurations = bucket.Properties.NotificationConfiguration.LambdaConfigurations;
```

Comment:
This could potentially break if the object doesn't have the structure we expect, could you use the `lodash/fp` [get](https://gist.github.com/jfmengels/6b973b69c491375117dc#_getpath-object) function instead?